CHANGELOG.md

```diff
@@ -1,91 +1,129 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [7.1.0]
+
+### Changed
+
+- Bypass numerical address cast ([#391](https://github.com/MetaMask/eth-sig-util/pull/391))
```

Comment:
```suggestion
- **BREAKING**: Values of type `number` are not accepted as address parameter anymore. Valid values are `string` and `Uint8Array`. ([#391](https://github.com/MetaMask/eth-sig-util/pull/391))
```
@OGPoyraz I think the change is breaking? Please confirm if this is right.

Reply:
You're right this message is not clear. [The removed part on this PR](https://github.com/MetaMask/eth-sig-util/pull/391) is removing the cast of the numerical address cast, I've adjusted the PR. 
I am a bit unsure that it should be a breaking change or not, what do you think?

Reply:
I would say it's breaking because:
- There exists input that was previously interpreted as valid and handled, which will now break
  - We can see example of this because of the need to remove these test cases: https://github.com/MetaMask/eth-sig-util/pull/391/files#diff-f6dbf9fbb07112db7776d1c764cea2d7e6df9d05cbdd800178d56ef5215a807bL4300-L4301

Reply:
Thanks @legobeat !
I've just spin up another release here, so closing this.